velox/common/base/tests/SimdUtilTest.cpp

```diff
@@ -107,9 +108,21 @@ class SimdUtilTest : public testing::Test {
     EXPECT_EQ(reference, target);
   }
 
+  void testSetAll() const {
```

Comment:
No need for creating a method in fixture.  Just create the test case and put the content there.